packages/client/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../../tsconfig",
+  "include": [
+    "src/*"
+  ]
+}
```

Comment:
Same as above.
All of the `tsconfig.json` below `packages` directory don't have a new line too.
Please revise all of them.